lib/grinder_files.dart

```diff
@@ -267,6 +268,8 @@ class FilePath {
 
   static String _coerce(arg) {
     if (arg is String) {
+      if (_sep != '/') arg = arg.replaceAll('/', _sep);
```

Comment:
I'll add some more docs. This would have the same behavior as the `getFile()` / `getDir()` methods late in the file. They'll work for relative paths, but it would not make sense for absolute ones. Those will likely be machine dependent anyway. (and auto-converting `/foo/bar` to `C:\foo\bar` would be difficult - we wouldn't know what drive to cvt to)